src/data/resolvers/motions.ts

```diff
@@ -425,11 +426,37 @@ export const motionsResolvers = ({
           },
         });
         if (data?.userReputation) {
-          const reward = await votingReputationClient.getVoterReward(
-            bigNumberify(motionId),
-            bigNumberify(data.userReputation),
-          );
-          return reward.toString();
+          let reward = bigNumberify(0);
+          let minReward = bigNumberify(0);
+          let maxReward = bigNumberify(0);
+          if (state === NetworkMotionState.Submit) {
```

Comment:
Why are you checking the state of the motion?

I don't believe the voter reward changes when the motion state changes? Does it?

Reply:
because i don't want to call function when its not needed?  performance wise